frontstage/views/surveys/help/surveys_help.py

```diff
@@ -187,13 +234,15 @@ def _send_new_message(subject, party_id, survey, business_id):
     return response
 
 
-def get_subject_and_breadcrumbs_title(option, uri):
-    """Gets the subject line for the secure message, the title of the breadcrumbs for sub options """
-    if option == 'do-not-have-specific-figures':
-        return 'I don’t have specific figures for a response', \
-               "Help completing this survey", \
-               "I don’t have specific figures for a response"
-    if option == 'unable-to-return-by-deadline':
-        return 'I’m unable to return the data by the deadline', \
-               "Help completing this survey", \
-               "I’m unable to return the data by the deadline"
+def _inside_legal_basis(legal_basis):
```

Comment:
This function name doesn't really describe what it's for.  I think this name (and the others in this PR cause there are a few) should be changed from `inside_legal_basis` to `is_legal_basis_mandatory` because the question this function is answering is whether the legal basis for the survey is for a mandatory survey.

I struggled to think of what it should be called that's short + describes what it is.... so if you think of something better then go for it!

Reply:
is_legal_basis_mandatory sounds good